src/services/accessValidation.js

```diff
@@ -0,0 +1,26 @@
+import { User, sequelize } from '../models';
+import logger from '../logger';
+
+const namespace = 'SERVICE:ACCESS_VALIDATION';
+
+/**
+ * Finds or creates a user in the database.
+ *
+ * //potentially use also or instead of the user id provided by HSES
+ * @param {Object} userData - user information containing email address, hses user id
+ * @returns {Promise<any>} - returns a promise
+ */
+export default function findOrCreateUser(data) {
```

Comment:
I couldn't find where this is called from. Is it used yet? Or here for future planned work?

Reply:
That's right. This is not connected yet. Wanted to include it, so that we have a function we could write tests against and verify the setup.